src/pages/vaults/VaultDepositWithdrawForm.tsx

```diff
@@ -90,7 +90,7 @@ export const VaultDepositWithdrawForm = ({
 }: VaultDepositWithdrawFormProps) => {
   const stringGetter = useStringGetter();
   const dispatch = useAppDispatch();
-  const { vaultsLearnMore, vaultTos } = useURLConfigs();
+  const { vaultLearnMore, vaultTos } = useURLConfigs();
```

Comment:
this isn't returned from `useUrlConfigs` yet right? Should we ensure that it has a value?

Reply:
Yep it was already there